@graphql-hive/core: CJS build requires `tslib` but declares it only in devDependencies

## Summary

`@graphql-hive/core` is compiled with `importHelpers: true`, so its published **CJS** build emits `require("tslib")`. But `tslib` is listed in `devDependencies`, not `dependencies` — so the published package requires a module it does not declare.

Reproduced on the current latest, `@graphql-hive/core@0.22.1`. The ESM build is unaffected.

```
$ grep -rl 'require("tslib")' cjs | wc -l
8
$ grep -rl 'tslib' esm | wc -l
0
$ node -e 'console.log(!!require("@graphql-hive/core/package.json").dependencies.tslib)'
false
```

Source: [`packages/libraries/core/package.json`](https://github.com/graphql-hive/console/blob/main/packages/libraries/core/package.json) has `tslib` under `devDependencies`, while the repo root [`tsconfig.json`](https://github.com/graphql-hive/console/blob/main/tsconfig.json) sets `"importHelpers": true` and `packages/libraries/core/tsconfig.json` extends it without overriding.

## Reproduction

Any resolver that does not hoist undeclared packages into scope. Minimal case with pnpm's hoisting disabled — no other trickery:

`package.json`
```json
{
  "name": "repro",
  "private": true,
  "dependencies": {
    "@graphql-hive/core": "0.22.1",
    "graphql": "16.14.2"
  },
  "packageManager": "pnpm@11.5.2"
}
```

`pnpm-workspace.yaml`
```yaml
packages:
  - "."
hoist: false
```

```
$ pnpm install --prod
$ node -e 'require("@graphql-hive/core")'
node:internal/modules/cjs/loader:1478
  throw err;
  ^
Error: Cannot find module 'tslib'
Require stack:
- .../node_modules/.pnpm/@graphql-hive+core@0.22.1_graphql@16.14.2/node_modules/@graphql-hive/core/cjs/index.js
```

The ESM entry loads fine in the same tree:

```
$ node --input-type=module -e 'import("@graphql-hive/core").then(() => console.log("ESM OK"))'
ESM OK
```

## Why this usually goes unnoticed

Three separate things mask it:

1. **npm / yarn** flat `node_modules` hoist `tslib` to the project root, so `require("tslib")` resolves from anywhere.
2. **pnpm** installs `tslib` anyway (it is a transitive dependency of `@graphql-tools/utils` among others) and symlinks it into the hidden hoist directory `node_modules/.pnpm/node_modules/tslib`, which *is* on the resolution path from `core`'s realpath. Default settings therefore also mask it.
3. Consumers mostly load the **ESM** build, which needs no helpers.

Node resolves `require("tslib")` from `@graphql-hive/core`'s own realpath, so the only directories consulted are `core`'s own dependency dir, the hidden hoist, and the app root. A consumer declaring `tslib` does not help `core`'s internal require — notably `@graphql-hive/cli@0.60.2` **does** declare `tslib: 2.8.1` and that does not prevent the failure, because the CLI's directory is not on `core`'s resolution path.

## How we hit it in production

`@graphql-hive/cli schema:publish` crashed in our deployed image with the stack above, because our CI shipped a `node_modules` tree in which the `.pnpm/node_modules/tslib` symlink was missing (a cache artifact on our side — our bug, not yours). Everything else about the tree was correct. Because `tslib` is undeclared, that single missing symlink was enough to break every schema publish, and the dependency graph gave no indication anything was wrong.

There is prior art for this failure class in this repo: #1075 (`Cannot find module 'tslib'`, same root cause in a different package).

## Suggested fix

Either:

- move `tslib` from `devDependencies` to `dependencies` in `packages/libraries/core/package.json` (matching what `@graphql-hive/cli` already does), or
- set `importHelpers: false` for this package's build so helpers are inlined into the CJS output and no runtime dependency exists.

The first is a one-line change and keeps output size unchanged.

Worth noting: because no source file imports `tslib` (the helpers are injected by the compiler), unused-dependency checkers report `tslib` as unused in a package that genuinely needs it at runtime — a plausible way it ended up in `devDependencies`, and worth a comment either way.

Happy to open a PR if a maintainer confirms the preferred option.

--- a/packages/libraries/core/package.json
+++ b/packages/libraries/core/package.json
@@ -54,7 +54,8 @@
     "events": "^3.3.0",
     "js-md5": "0.8.3",
     "lodash.sortby": "^4.7.0",
-    "tiny-lru": "^11.4.7"
+    "tiny-lru": "^11.4.7",
+    "tslib": "^2.8.1"
   },
   "devDependencies": {
     "@apollo/federation": "0.38.1",

